CHANGELOG.md

```diff
@@ -4,6 +4,73 @@
+* [#7854](https://github.com/timescale/timescaledb/pull/7854) Remove support for PG14
+* [#7864](https://github.com/timescale/timescaledb/pull/7854) Allow adding CHECK constraints to compressed chunks
+* [#7868](https://github.com/timescale/timescaledb/pull/7868) Allow adding columns with `CHECK` constraints to compressed chunks
+* [#7874](https://github.com/timescale/timescaledb/pull/7874) Support for SkipScan for distinct aggregates over the same column
+* [#7877](https://github.com/timescale/timescaledb/pull/7877) Remove blocker for unique constraints with `ADD COLUMN`
+* [#7878](https://github.com/timescale/timescaledb/pull/7878) Don't block non-immutable functions in continuous aggregates
+* [#7880](https://github.com/timescale/timescaledb/pull/7880) Add experimental support for window functions in caggs
+* [#7899](https://github.com/timescale/timescaledb/pull/7899) Vectorized decompression and filtering for boolean columns
+* [#7915](https://github.com/timescale/timescaledb/pull/7915) New option `refresh_newest_first` to CAgg refresh policy API
+* [#7917](https://github.com/timescale/timescaledb/pull/7917) Remove `_timescaledb_functions.create_chunk_table` function
+* [#7929](https://github.com/timescale/timescaledb/pull/7929) Add `CREATE TABLE ... WITH` API for creating hypertables
+* [#7946](https://github.com/timescale/timescaledb/pull/7946) Add support for splitting a chunk
+* [#7958](https://github.com/timescale/timescaledb/pull/7958) Allow custom names for user-defined actions
+* [#7972](https://github.com/timescale/timescaledb/pull/7972) Add vectorized filtering for constraint checking while backfilling into compressed chunks
+* [#7976](https://github.com/timescale/timescaledb/pull/7976) Include Continuous Aggregate name in jobs informational view.
+* [#7977](https://github.com/timescale/timescaledb/pull/7977) Replace references to compression with columnstore
+* [#7981](https://github.com/timescale/timescaledb/pull/7981) Add columnstore as alias for enable_columnstore in `ALTER TABLE`
+* [#7983](https://github.com/timescale/timescaledb/pull/7983) Support for SkipScan over compressed data
+* [#7991](https://github.com/timescale/timescaledb/pull/7991) Improves default segmentby options
+* [#7992](https://github.com/timescale/timescaledb/pull/7992) Add API into hypertable invalidation log
+* [#8000](https://github.com/timescale/timescaledb/pull/8000) Add primary dimension info to information schema
+* [#8005](https://github.com/timescale/timescaledb/pull/8005) Support `ALTER TABLE SET (timescaledb.chunk_time_interval='1 day')`
+* [#8012](https://github.com/timescale/timescaledb/pull/8012) Add event triggers support on chunk creation
+* [#8014](https://github.com/timescale/timescaledb/pull/8014) enable bool compression by default by setting `timescaledb.enable_bool_compression=true`. Note: for downgrading to `2.18` or earlier version one needs to use the provided downgrade script in 'timescaledb-extras'.
```

Comment:
We can add a link pointing to the script. WDYT?

Reply:
Yes, good call. I'll add the link

Reply:
done.